artifacts/deploy/bootstrap-token-configuration.yaml

```diff
@@ -203,3 +188,79 @@ subjects:
 - apiGroup: rbac.authorization.k8s.io
   kind: Group
   name: system:nodes
+
+# ---
+# apiVersion: rbac.authorization.k8s.io/v1
+# kind: Role
+# metadata:
+#   name: system:karmada:agent-secret
+#   namespace: "{{cluster_namespace}}" # default to karmada-cluster
+# rules:
+# - apiGroups:
+#   - ""
+#   resources:
+#   - secrets
+#   # resourceNames:
+#   # - {{clustername}}-impersonator
```

Comment:
```suggestion
#   # - {{clustername}}-impersonator
#   # - {{clustername}}
```
refer to
https://github.com/karmada-io/karmada/blob/07c6cb4db9d52805a1f8043912849bcec9ac5f8b/pkg/util/credential.go#L169-L187


agent needs the rbac permission for secrets `{{clustername}}-impersonator` and `{{clustername}}`

Reply:
Thanks! Fixed.